apps/web/src/features/finance/actions/getPayments.ts

```
@@ -0,0 +1,643 @@
+ * The view simplifies filtering by client_id and product_id by flattening the
+ * payment -> payment_slots -> payment_plans -> clients/products relationship.
+ */
+
+export async function getPayment(id: string) {
+  try {
+    const supabase = await createClient();
+
+    // Fetch payment with related data from view
+    const { data: row, error } = await supabase
+      .from("payments_with_details") // Using 'as any' temporarily until view is created
+      .select(`*`)
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)